### PR TITLE
Test the error boundary changes

### PR DESCRIPTION
**Pull request by tuckerzp**
This tests changes introduced by https://github.com/EasyDynamics/oscal-react-library/pull/490

### CODE REVIEW

**Comment by tuckerzp** [comment]
This will fail until #113 is merged in.

**Comment by tuckerzp** [comment]
It seems like `Drawer.cy.js` testing is failing. Which would not be touched by these tests added, but that does not seem fantastic. I will try to look into why they might be failing.

**Comment by tuckerzp** [comment]
**UPDATE**: Rerunning the tests now pass. Having the tests randomly fail is definitely not great. Should I create an issue to track this?

**Comment by kylelaker** [comment]
> Should I create an issue to track this

I don't want to do that for something we've only seen once. If this happens multiple times, then probably.

**Review by laurelmay — APPROVED** [review]
Good stuff! This should prevent future regressions on this.

File: end-to-end-tests/cypress/e2e/OSCAL/OSCAL_Load_Tests.cy.js [hunk 1]
```diff
@@ -37,12 +37,26 @@ describe('Test Loading Component Definitions', () => {
...
+  });
+
+  it('display "yikes" on load of wrong object type', () => {
     cy.contains('Yikes').should('be.visible')
-  })
-})
\ No newline at end of file
+  });
+
+  it('do not persist after loading a valid component in Viewer', () => {
```
Comment (mikeisen1) [inline end-to-end-tests/cypress/e2e/OSCAL/OSCAL_Load_Tests.cy.js:52]: Maybe I am misunderstanding something, but why would there be an issue of persistence here or after loading a valid component?

Comment (tuckerzp) [reply end-to-end-tests/cypress/e2e/OSCAL/OSCAL_Load_Tests.cy.js:52]: You are right that there shouldn't be. https://github.com/EasyDynamics/oscal-react-library/pull/490 fixed it and this test is to make sure we do not reintroduce that error. 

Essentially if I loaded a bad `component` in the viewer I would get the standard error message. Typing in the url of a valid `Component` and then hitting `Reload` should make that error go away. It was not doing that.

**Comment by mikeisen1** [comment]
@tuckerzp, there is a conflict that should be resolved before we can merge this PR.

**Review by mikeisen1 — APPROVED** [review]

**Pull request merged**